orchagent/port_rates.lua

```diff
@@ -0,0 +1,88 @@
+-- KEYS - port IDs
+-- ARGV[1] - counters db index
+-- ARGV[2] - counters table name
+-- ARGV[3] - poll time interval
+-- return log
+
+local logtable = {}
+
+local function logit(msg)
+  logtable[#logtable+1] = tostring(msg)
+end
+
+local counters_db = ARGV[1]
+local counters_table_name = ARGV[2] 
+local rates_table_name = "RATES"
+
+-- Get configuration
+redis.call('SELECT', counters_db)
+local smooth_interval = redis.call('HGET', rates_table_name .. ':' .. 'PORT', 'PORT_SMOOTH_INTERVAL')

```

Comment:
>: [](start = 64, length = 1)

The table separator are from database_config.json. Do not hardcode.

Reply:
It's not trivial to access the database config from the lua script. Should we pass the separator as one of the parameters?
I see the separator hardcoded in a lot of other places, like [this](https://github.com/Azure/sonic-swss/blob/master/orchagent/pfc_restore.lua#L18). If there is a need to avoid such things in future, I suggest separate PR and refactor it in all the lua scripts. 

Some scripts have other dbs/table names hardcoded, and ATM receive the same ARGV. Need to suggest broad solution.

Reply:
Lua parameter is good enough.
Could you please add a TODO comment or Github issue?

Reply:
@qiluo-msft Created issue [#1319](https://github.com/Azure/sonic-swss/issues/1319)